test/testcafe/spec/EnrollProfileUpdateView_spec.js

```diff
@@ -65,4 +74,25 @@ test.requestHooks(requestLogger, xhrEnrollProfileUpdateAllOptionalMock)('should
   await enrollProfileUpdatePage.clickSetUpSkipLink();
   const req = requestLogger.requests[0].request;
   await t.expect(req.url).eql('http://localhost:3000/idp/idx/skip');
-});
\ No newline at end of file
+});
+
+test.requestHooks(requestLogger, xhrEnrollProfileUpdateAllOptionalSuccess)('should set secondEmail default to empty on form submit when all fields are optional', async t => {
+  const enrollProfileUpdatePage = new EnrollProfileUpdateViewPageObject(t);
+  const identityPage = await setup(t);
+  await identityPage.fillIdentifierField('test');
+  await identityPage.fillPasswordField('test 123');
+  await identityPage.clickNextButton();
+
+  requestLogger.clear();
+  await enrollProfileUpdatePage.clickFinishButton();
+  const req = requestLogger.requests[0].request;
+  await t.expect(req.url).eql('http://localhost:3000/idp/idx/enroll/new');
+  const reqBody = JSON.parse(req.body);
+  await t.expect(reqBody).eql({
+    userProfile: {
+      secondEmail: '',
+    },
+    stateHandle: '02m5bP1_PZfgeox0qFcNeh5Zr71MB3gs3tw7kCsL2G',
+  });
```

Comment:
Test the request body when there are required fields?

Reply:
where there are required field it's the default behavior and it always exists in payload like `stateHandle`.. No need to test that IMO